sequelize/db.test.js

```diff
@@ -0,0 +1,41 @@
+jest.mock('sequelize')
+const {Sequelize} = require('sequelize')
+const {start, update, close, Account} = require('./db')
+
```

Comment:
had `require('winston')` and `jest.mock('winston')` set up here causing the error i sent.